rpm-ostree-bisect

```diff
@@ -97,6 +97,39 @@ def log(msg):
     print(msg)
     sys.stdout.flush()
 
+"""
+    Find out of the given commitid is a base commit (i.e. no
+    layered packages). We'll cue off of the `rpmostree.clientlayer`
+    metadata for this.
+"""
+def is_base_commit(repo, commitid):
+    # Grab commit object. If None then history has been
+    # trimmed from the remote and we can break
+    _, commit = repo.load_variant_if_exists(
+                    OSTree.ObjectType.COMMIT, commitid)
+    # Grab version info from commit
+    meta = commit.get_child_value(0)
+    clientlayer = meta.lookup_value('rpmostree.clientlayer', GLib.VariantType.new('b'))
+    if clientlayer:
+        return False
+    else:
+        return True
```

Comment:
Hmm, feels like it'd be cleaner to just call this function `is_layered_commit` and here `return meta.lookup_value(...)` directly.

Reply:
ok will change it